products/samsungmobile.md

```diff
@@ -153,6 +153,11 @@ releases:
     eol: false
     releaseDate: 2021-03-12
 
+-   releaseCycle: "Galaxy Xcover 4"
+    support: false
+    eol: 2020-11-19
```

Comment:
I don't find this date in your mentioned links. I would change it to `2019-12-01` as last patch date in your screenshot.